### PR TITLE
Spelling: The location, days to keep x3, commas, against filemask, inexistant

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate/addons/removal.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ class RemoveSuggestions(RemovalAddon):
     name = 'weblate.removal.suggestions'
     verbose = _('Remove old suggestions')
     description = _(
-        'Remove suggestions without votes older than given age in days.'
+        'Days to keep suggestions without votes.'
```
Comment (nijel) [inline weblate/addons/removal.py]: This is long description of the addon, your text doesn't make sense there (samoe for description above):

Screenshot_2019-05-11 Weblate Application

File: weblate/addons/forms.py [hunk 2]
```diff
@@ -200,7 +200,7 @@ class DiscoveryForm(BaseAddonForm):
         initial='po',
         required=True,
         help_text=_(
-            'Automatic detection might fail for some formats '
+            'Automatic detection might fail for some formats, '
             'and is slightly slower.'
```
Comment (nijel) [inline weblate/addons/forms.py]: Hmm, there is no automatic detection anymore, so this needs more than grammar fixes. I'll adjust this one...

File: weblate/addons/forms.py [hunk 3]
```diff
@@ -171,7 +171,7 @@ class JSONCustomizeForm(BaseAddonForm):
 
 class RemoveForm(BaseAddonForm):
     age = forms.IntegerField(
-        label=_('Threshold age for removal in days'),
+        label=_('Days to keep'),
```
Comment (comradekingu) [inline weblate/addons/forms.py]:
```suggestion
        label=_('Timeframe for removal'),
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**